docs/semgrep-appsec-platform/gitlab-mr-comments.md

```diff
@@ -95,6 +95,13 @@ Creating a PAT grants the API scope to Semgrep, which lets it post comments.
 
 For more configuration options, see [GitLab CI Sample](/semgrep-ci/sample-ci-configs#gitlab-cicd).
 
+:::info MR comments with multiple GitLab groups
+If you're using Semgrep with multiple GitLab groups, ensure that you've completed one of the following steps to ensure that you see MR comments for repositories associated with all of your groups:
+
+1. Create a Semgrep account connection with each GitLab group by signing in to your Semgrep account, navigating to **Settings > Source code managers**, and adding an entry for each group.
```

Comment:
Since gitlab supports the concept of sub-groups, perhaps we might want to mention that you only need to add a single entry for the parent group when you want to onboard ALL subgroups of a parent group.